### DISCUSSION TITLE
Update on API Food Asset Map

### DISCUSSION THREAD

**Discussion by dgkeyes** [rfortherestofus/api-food-asset-map#23] [General]
Hi @katiejolly and @grahamammal! Thanks so much for your help getting all of the data for this project. 

Looking at the issues in the Import Data milestone, I think we're getting close. @grahamammal I know you said you should be able to wrap up your issues this week. @katiejolly will you be able to do the same by the end of the weekend, say? 

@katiejolly as we move on to building out the map, do you think you and I could check in briefly early next week? Just want to 1) make sure we have all the data imported, 2) make sure you have everything you need, and 3) discuss how you're planning to build out the map + distill website. Please feel free to schedule with [this link](https://savvycal.com/davidkeyes/02778d8d).

Thanks again to both of you!

**Comment by katiejolly** [comment]
Yep, end of the weekend sounds good!

**Reply by dgkeyes** [reply]
Great, thanks, Katie!

**Reply by dgkeyes** [reply]
I just took a quick look at the data that's been imported. I think between you, @grahamammal, and me, we've added all of the various types of data we initially. @katiejolly if you don't mind checking to see if everything looks ok and you have what you need for the mapping, that would be great. I'm thinking specifically about the types of providers, as defined by the client: 

- Providers that offer food pantries
- Providers that offer free, prepared food/hot meals
- Food banks
- Food pharmacies
- Drug stores
- Ethnic food markets
- Full service grocery stores/supermarkets/markets (non SNAP or WIC)
- Stores that accept SNAP/WIC (full service grocery stores, supermarkets, markets, ethnic food markets, Target, Walmart)
- Corner stores
- Fast food restaurants
- Restaurants
- Farmers markets
- Liquor Stores

For some of these, we'll need to do some additional wrangling. For example, to determine whether stores are in the SNAP/WIC accepting category, we'll need to join the grocery store data with the SNAP/WIC data. I'm guessing matching might be tough due to inconsistencies in names, etc. If there are things that Ellen can work on here to help you with the mapping, please let us know. 

And, just in general, if there are any other tweaks we need to make to the data now that Ellen can help with, please just shout. Thanks!

**Reply by dgkeyes** [reply]
Just sent an email to you @grahamammal about helping out again with this project. [I made a video to show a broad overview of where I'm hoping the data ends up](https://share.getcloudapp.com/jkuLpwJg). Please let me know if this makes sense. 

I think it will make most sense to do this work in a separate branch because @katiejolly is going to build out a prototype of the map in the main branch. 

I'll also go ahead and add explanations to the various issues I've assigned you to. 

Please let me know if you have any questions. Thanks!
